content/es/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,100 @@
+
+
+Los componentes del control plane se comunican con el servidor API a través del puerto seguro.
+
+Como resultado, el modo de operación predeterminado para las conexiones desde los nodos y el pod que se ejecutan en los nodos hacia el plano de control está asegurado por defecto y puede funcionar sobre redes no confiables y/o públicas.
+
+## Control plane al nodo
+
+Existen dos formas de comunicación primarias desde el control plane (el servidor API) hacia los nodos. El primero es desde el servidor API hasta el proceso {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} que se ejecuta en cada nodo del clúster. El segundo es desde el servidor API hacia cualquier nodo, pod o servicio a través de la funcionalidad de proxy del servidor API.
+
+
+### Servidor API a kubelet
+
+Las conexiones desde el servidor API al kubelet se utilizan para:
+
+* Obtener registros de los pods.
+* Conectarse (generalmente a través de `kubectl`) a pods en ejecución.
+* Proporcionar la funcionalidad de reenvío de puertos del kubelet.
+
+Estas conexiones terminan en el punto final HTTPS del kubelet. Por defecto, el servidor API no verifica el certificado de servicio del kubelet, lo que hace que la conexión esté sujeta a ataques de intermediario y sea insegura para ejecutarse sobre redes no confiables y/o públicas.
+
+Para verificar esta conexión, usa la bandera `--kubelet-certificate-authority` para proporcionar al servidor API un paquete de certificados raíz para usar en la verificación del certificado de servicio del kubelet.
+
+
+Si eso no es posible, usa [túneles SSH](#ssh-tunnels) entre el servidor API y el kubelet si es necesario para evitar conectarse a través de una red no confiable o pública.
+
+Finalmente, se debe habilitar la [autenticación y/o autorización del Kubelet ](/docs/reference/access-authn-authz/kubelet-authn-authz/) para asegurar API del servicio kubelet  
+
+
+### Servdor API a nodes, pods, y services
+
+
+Las conexiones desde el servidor API a un nodo, pod o servicio por defecto son conexiones HTTP simples y, por lo tanto, no están autenticadas ni cifradas. Pueden ejecutarse sobre una conexión HTTPS segura anteponiendo `https:` al nombre del nodo, pod o servicio en la URL de la API, pero no validarán el certificado proporcionado por el punto final HTTPS ni proporcionarán credenciales de cliente. Así que, aunque la conexión estará cifrada, no ofrecerá garantías de integridad. Estas conexiones **actualmente no son seguras** para ejecutarse sobre redes no confiables o públicas.
+
+
+### Tuneles SSH
+
+
+Kubernetes soporta túneles SSH para proteger los caminos de comunicación del plano de control a los nodos. En esta configuración, el servidor API inicia un túnel SSH a cada nodo en el clúster (conectándose al servidor SSH que escucha en el puerto 22) y pasa todo el tráfico destinado a un kubelet, nodo, pod o servicio a través del túnel.
```

Comment:
Previously in the above paragraphs, we have used _"control plane"_ without translating. So, you may want to use the words in English to keep consistency. 

```suggestion
Kubernetes soporta túneles SSH para proteger los caminos de comunicación del control plane a los nodos. En esta configuración, el servidor API inicia un túnel SSH a cada nodo en el clúster (conectándose al servidor SSH que escucha en el puerto 22) y pasa todo el tráfico destinado a un kubelet, nodo, pod o servicio a través del túnel.
```